chapters/03_queueing_model/06_closed_form_steady_state_probs/main.tex

```diff
@@ -0,0 +1,1022 @@
+\subsubsection{Closed-form approach}
+
+This section aims to investigate the connection between Markov chains and graph
+theory by exploring alternative ways of calculating the steady state
+probabilities and considering the problem from different perspectives.
+
+\paragraph{Parameters}
+The parameters considered as inputs are:
+\begin{itemize}
+    \item the number of servers \(C\),
+    \item the threshold \(T\),
+    \item the capacity of node 1 \(N\),
+    \item the capacity of node 2 \(M\).
+\end{itemize}
+
+Additional parameters of the model are the type 1 individuals arrival rate,
+the type 2 individuals arrival rate and the service rate (\(\lambda_1,
+\lambda_2, \mu\)).
+More specifically, the way these parameters are translated into the model are:
+
+\begin{itemize}
+    \item \textbf{Number of servers (\(C\)):} Affects the weight of all edges
+    \((v_i, v_j) \in E\) in the Markov chain that correspond to a service rate.
+    These edges have a weight of:
+    \begin{equation*}
+        w_{(v_i, v_j)} = q_{v_i, v_j}
+    \end{equation*}
+    where \(q_{i,j}\) is defined in equation~(\ref{eq:markov_transition_rate}).
+    Thus, the coefficients of the service rate have a lower bound of \(1\) and
+    an upper bound of \(C\).
+    \item \textbf{Threshold (\(T\)):} Determines the length of the left
+    \textit{arm} of the model.
+    In essence the threshold acts as a breakpoint between states where \(u=0\)
+    and states where \(0 \leq u \leq M\).
+    Increasing \(T\) results in having more set of states where \(u\) can only
+    be \(0\).
+    \item \textbf{Node 1 capacity (\(N\)):} Is the upper bound of \(v\)
+    for all  states \((u,v)\).
+    \item \textbf{Node 2 capacity (\(M\)):} Is the upper bound of \(u\)
+    for all states \((u,v)\) such that \(v \geq T\).
+\end{itemize}
+
+
+\paragraph{Example figure of Markov Model}
+
+\begin{figure}[H]
+    \centering
+    \scalebox{0.7}{
+        \input{chapters/03_queueing_model/img/example_model_1352/main.tex}
+        }
+    \caption{\(C=1, T=3, N=5, M=2\)}
+    \label{fig:Markov_1352_example_for_closed_form}
+\end{figure}
+
+In Figure~\ref{fig:Markov_1352_example_for_closed_form} an example of such a
+Markov model is shown where \(C=1\), \(T=3\) which means that the \textit{left
+arm} of the model has a length of \(3\), \(N=5\) that indicates that the
+right-most states \((u,v)\) are of the form \((u,5)\) and \(M=2\) that
+equivalently shows that the bottom states are of the form \((2,v)\).
+
+\paragraph{A graph theoretic model underlying the Markov chain}
+
+An additional approach that one may consider to get the state probabilities is
+the graph theoretical approach for state probabilities.
+Thus, it can be assumed that a Markov chain model \(M\) can be translated as a
+weighted directed graph \(G_M = (V, E)\) where \(V=S\) from
```

Comment:
I think this needs to come before the previous section.

I don't think the wording is correct either, it's not "Thus," but it can just be assumed.

Reply:
Yeah. I think formally defining the graph $G=(V,E)$ has to come before anything else.